src/System.Management.Automation/engine/CultureVariable.cs

```diff
@@ -25,7 +25,7 @@ public override object Value
             get
```

Comment:
@iSazonov, your last commit had 36 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:15 char:37
+                 @{ Script = { $true ? 1 : 2 };  ExpectedValue = 1 }
+                                     ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:16 char:38
+                 @{ Script = { $true? ?1 :2 };   ExpectedValue = 2 }
+                                      ~~
Unexpected token '?1' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:17 char:38
+                 @{ Script = { ${true}?1:2 };    ExpectedValue = 1 }
+                                      ~~~~
Unexpected token '?1:2' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:18 char:33
+                 @{ Script = { 1 ? 1kb : 0xf };  ExpectedValue = 1kb }
+                                 ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:19 char:33
+                 @{ Script = { 0 ?1kb:0xf };     ExpectedValue = 15 }
+                                 ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:20 char:35
+                 @{ Script = { 's' ?1kb:0xf };   ExpectedValue = 1kb }
+                                   ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:21 char:37
+                 @{ Script = { $null ?1kb:0xf }; ExpectedValue = 15 }
+                                     ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:22 char:34
+                 @{ Script = { '' ?1kb:0xf };    ExpectedValue = 15 }
+                                  ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:25 char:39
+                 @{ Script = { 1,2,3,4 ? 'Core' : 'Desktop' };         ...
+                                       ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:26 char:42
+                 @{ Script = { @(1,2,3,4) ? 'Core' : 'Desktop' };      ...
+                                          ~
Unexpected token '?' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Set/New/Remove-Service cmdlet tests.New-Service can create a new service called 'testautomatic'
```powershell
Expected 'one', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 217
217:             $service.displayname | Should -Be $(if($displayname){$displayname}else{$name})
```
Set/New/Remove-Service cmdlet tests.New-Service can create a new service called 'testmanual'
```powershell
Expected 'two', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 217
217:             $service.displayname | Should -Be $(if($displayname){$displayname}else{$name})
```
Set/New/Remove-Service cmdlet tests.New-Service can create a new service called 'testdisabled'
```powershell
Expected 'testdisabled', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 217
217:             $service.displayname | Should -Be $(if($displayname){$displayname}else{$name})
```
WSMan Config Provider.Set-Item tests.Set-Item on non-existent property should fail
```powershell
Expected an exception, with FullyQualifiedErrorId 'System.InvalidOperationException,Microsoft.PowerShell.Commands.SetItemCommand' to be thrown, but the FullyQualifiedErrorId was 'WsManError'. from D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1:213 char:15
    +             { Set-Item $testPluginPath\foo "bar" } | Should -Throw -E ...
    +               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.WSMan.Management\ConfigProvider.Tests.ps1: line 213
213:             { Set-Item $testPluginPath\foo "bar" } | Should -Throw -ErrorId "System.InvalidOperationException,Microsoft.PowerShell.Commands.SetItemCommand"
```